src/modules/dashboard/components/ColonyHomeActions/ColonyHomeActions.tsx

```diff
@@ -25,7 +25,7 @@ import ManageReputationDialog from '~dialogs/ManageReputationDialog';
 import ColonyTokenManagementDialog from '~dialogs/ColonyTokenManagementDialog';
 import { SmiteDialog, AwardDialog } from '~dialogs/AwardAndSmiteDialogs';
 import ManageSafeDialog from '~dialogs/ManageSafeDialog';
-import ControlSafeDialog from '~dialogs/ControlSafeDialog';
+import ControlSafeDialog from '~dialogs/ControlSafeDialog/ControlSafeDialog';
```

Comment:
After rebasing, I received this error that `ControlSafeDialog` couldn't be found in the index file of its directory. Quick fix to get around this was to link directly to the component file. But shouldn't have to, really. Could the next reviewer confirm also that trying to import straight from the index file breaks the bundle?

Reply:
Yes, this is also breaking for me when `import ControlSafeDialog from '~dialogs/ControlSafeDialog';` is used and I get the following TypeError:
 
<img width="549" alt="Screen Shot 2022-10-27 at 9 44 01 PM" src="https://user-images.githubusercontent.com/71563622/198477588-7a34b0df-bce9-4fe5-a479-be44751d5e11.png">

But everything works well with `import ControlSafeDialog from '~dialogs/ControlSafeDialog/ControlSafeDialog';`

Reply:
I can confirm as well, it's really weird